wgengine/userspace.go

```diff
@@ -794,7 +801,7 @@ func (e *userspaceEngine) Reconfig(cfg *wgcfg.Config, routerCfg *router.Config,
 	if err := e.magicConn.SetPrivateKey(wgkey.Private(cfg.PrivateKey)); err != nil {
 		e.logf("wgengine: Reconfig: SetPrivateKey: %v", err)
 	}
-	e.magicConn.UpdatePeers(peerSet)
```

Comment:
where did this go?

Reply:
oops. restored. Worrying that no tests failed.